app/v3/retrievePeriodSummary/RetrievePeriodSummaryConnector.scala

```diff
@@ -59,7 +59,7 @@ class RetrievePeriodSummaryConnector @Inject() (val http: HttpClient, val appCon
         result
```

Comment:
Remove this feature switch `featureSwitches.isDesIf_MigrationEnabled` with `featureSwitches: SeBusinessFeatureSwitches` thanks. It is already enabled in [production](https://github.com/hmrc/app-config-production/blob/main/self-employment-business-api.yaml#L67) so should be removed as part of these changes. Also, remove `isDesIf_MigrationEnabled` entirely, thanks.